packages/react/src/Card/CardHeadingGroup.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { render, screen } from '@testing-library/react'
+import { createRef } from 'react'
+import { CardHeadingGroup } from './CardHeadingGroup'
+import '@testing-library/jest-dom'
+
+describe('CardHeadingGroup', () => {
+  it('renders', () => {
+    const { container } = render(<CardHeadingGroup tagline="test" />)
+
+    const component = container.querySelector(':only-child')
```

Comment:
Can we query by the role of ‘group’? https://www.w3.org/TR/html-aria/#el-hgroup

Reply:
Hm, that's odd. It doesn't show up as role `group` in Chrome's a11y tree. Does it for you?

Reply:
You’re right, it doesn’t. Does our test runner use Chromium as well?

Reply:
No, it uses Jest with JSDOM, which gives you a subset of the DOM I think? I'm not sure, it's a little black box-y. 

But it's easy to test. When I use `screen.getByRole('group')`, Testing Library is unable to find the element, so the spec doesn't seem to be implemented in Jest / JSDOM either.